service/src/test/java/bio/terra/policy/service/pao/PaoUpdateTest.java

```diff
@@ -476,6 +477,34 @@ void updateOneSourceTest_sourcePolicyPropagatesCorrectly() throws Exception {
     assertTrue(checkD.getEffectiveAttributes().getInputs().isEmpty());
   }
 
+  @Test
```

Comment:
I copied this over to `main` and confirmed that it ran successfully (minus the `assertThrows` at the bottom) and that it would've caught the regression discussed in this [comment thread](https://github.com/DataBiosphere/terra-policy-service/pull/101#discussion_r1657725332)